concourse/scripts/generate-cf-secrets.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+set -x
+
+SCRIPTS_DIR=`dirname $0`
+OUTPUT_FILE=$1
+
+pwgen() {
+   openssl rand -base64 12 | sed 's/+/0/;s/\//0/'
```

Comment:
Actually, this won't work in the case of repeated `+` or `/`. Can you please add the modifier `g` to sed:

`sed 's/+/0/g;s/\//0/g'`